src/tex/wandw-ch12.tex

```diff
@@ -3,147 +3,163 @@
-
-by \hardsectionref{4}{7}, and put z = l after the differentiations have been
-performed.]
-
-Example 2. Shew that
-
-,11 1 P 1
-
-2 3 n Jo
-
-t and hence that Euler's constant y is given by||
-
--(1-""*,
-
-lim
-
-[/: '-04)" ?-/:(-r.)"T']'
-
-Example 3. Shew that
-
-e> r(g + l)
-
-T z-x + ) '
-
-n = \ \ \ Z + n) J
-
-* Journal filr Math. li. (1856). This formula for F(z) had been
-obtained from Euler's formula \hardsubsectionref{12}{1}{1}) in 1848 by F. W. Newman,
-Cambridge and Dublin Math. Journal, iii. (1848), p. 60. t Math. Ann.
-sxvm. (1887), pp. 1-13. + Math. Ann. xlviii. (1897), pp. 70-74. §
-Messenger of Math. xxix. (1900), pp. 122-128. li The reader will see
-later \hardsectionref{12}{2} example 4) that this limit may be written
+\begin{displaymath}
+  \frac{1}{\Gamma(z)} = ze^{\gamma z} \prod_{n=1}^\infty
+  \thebrace{\theparen{1 + \frac{z}{n}}  e^{-\frac{z}{n}}} ;
+\end{displaymath}
+\index{Weierstrassian product} from this equation it is apparent that
+$\Gamma(z)$ \emph{is analytic except at the points
+  $z=0, -1, -2, \ldots$, where it has simple poles}.
+
+Proofs have been published by
+H\"older$^\dagger$\footnote{\emph{Math. Ann.}  \textsc{XXVIII.}
+  (1887), pp. 1-13.}, % TODO footnote/reference
+Moore$^\ddagger$\footnote{\emph{Math. Ann.} \textsc{XLVIII.} (1897),
+  pp. 70-74.}, % TODO footnote/reference
+and Barnes$^\S$\footnote{\emph{Messenger of Math.} \textsc{XXIX}. (1900),
+  pp. 122-128.} % TODO footnote/reference
+of a theorem known to Weierstrass \index{Weierstrass' theorem on
+  Gamma-functions} that the Gamma-function does not satisfy any
+differential equation with rational coefficients.
+
+%Example 1.
+\begin{wandwexample}
+  Prove that
+
+  \begin{displaymath}
+    \Gamma(1) = 1, \qquad \Gamma'(1)=-\gamma,
+  \end{displaymath}
+
+  where $\gamma$ is Euler's constant.
+
+  [Justify differentiating logarithmically the equation
+  \begin{displaymath}
+    \frac{1}{\Gamma(z)} = ze^{\gamma z} \prod_{n=1}^\infty
+    \thebrace{\theparen{1 + \frac{z}{n}}  e^{-\frac{z}{n}}} 
+  \end{displaymath}
+  by \hardsectionref{4}{7}, and put $z = 1$ after the differentiations
+  have been performed.]
+\end{wandwexample}
+
+%Example 2. 
+\begin{wandwexample}
+  Shew that
+  \begin{displaymath}
+    1 + \half + \frac{1}{3} + \ldots + \frac{1}{n} =
+    \int_0^1 \frac{1 - (1 - t)^n}{t} \dmeasure t ,
+  \end{displaymath}
+  and hence that Euler's constant $\gamma$ is given by$^{||}$ \footnote{The
+    reader will see later \hardsectionref{12}{2} example 4) that this
+    limit may be written 
+    \begin{displaymath}
+      \int_0^1 \theparen{1 - e^{-t}} \frac{\dd t}{t} - 
+      \int_1^\infty \frac{e^{-t} \dmeasure t}{t} .
+    \end{displaymath}
+  } % TODO footnote
+  \begin{displaymath}
+  \lim_{n \rightarrow \infty} \thebracket{\int_0^1 \thebrace{1 -
+      \theparen{1-\frac{t}{n}}^n}  \frac{\dmeasure t}{t} - 
+      \int_1^n \theparen{1-\frac{t}{n}}^n  \frac{\dmeasure t}{t}} .
+  \end{displaymath}
```

Comment:
It would seem my early-morning mental parser was faulty. My apologies for the confusion! Thank you again.